### PR TITLE
Add a caption to my picture

### PR DESCRIPTION
**Pull request by williamgonzalez9060**
Pull request by Wg

### CODE REVIEW

File: _posts/0000-01-03-williamgonzalez9060.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
----
+﻿---
 layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _posts/0000-01-03-williamgonzalez9060.md:3]: this is bland.

File: _posts/0000-01-03-williamgonzalez9060.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 ---
 
 ![chill-out-lemur](https://cloud.githubusercontent.com/assets/16547949/25400761/9c64ee82-29c1-11e7-8ad3-6886f97634b9.jpg)
 {: .center}
 
-Caption here
+When is lunch?
```
Comment (beardofedu) [inline _posts/0000-01-03-williamgonzalez9060.md:9]: as soon as the instructor figures out what he is doing

**Review by beardofedu — APPROVED** [review]
This is a general review summary. General.

**Review by donnabenesh — COMMENTED** [review]
looks good to me

**Pull request merged**